sdk/identity/azure-identity/README.md

```diff
@@ -209,7 +209,7 @@ client = EventHubClient(host, event_hub_path, credential_chain)
 This library includes an async API supported on Python 3.5+. To use the async
 credentials in [azure.identity.aio][ref_docs_aio], you must first install an
 async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). See
-[azure-core documentation](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/core/azure-core/README.md#transport)
+[azure-core documentation](../../core/azure-core/README.md#transport)
```

Comment:
These links are absolute because the readme is baked into the package and becomes its description at [PyPI](https://pypi.org/project/azure-identity/).

Reply:
We rewrite the relative links to absolute links see https://github.com/Azure/azure-sdk-for-python/pull/8813#discussion_r349817338.